src/processTargets/targets/WeakTarget.ts

```diff
@@ -21,6 +22,12 @@ export default class WeakTarget extends BaseTarget {
     return true;
   }
 
+  getEditNewContext(_isBefore: boolean): EditNewContext {
+    return {
+      delimiter: "\n",
```

Comment:
shouldn't we just use a final stage of weak line in `EditNew` to convert a weak target to a line?

Reply:
Sounds good